addons/sale_mrp/models/sale.py

```diff
@@ -115,7 +115,7 @@ def _compute_qty_delivered(self):
                     }
                     order_qty = order_line.product_uom._compute_quantity(order_line.product_uom_qty, relevant_bom.product_uom_id)
                     qty_delivered = moves._compute_kit_quantities(order_line.product_id, order_qty, relevant_bom, filters)
-                    order_line.qty_delivered = relevant_bom.product_uom_id._compute_quantity(qty_delivered, order_line.product_uom)
+                    order_line.qty_delivered += relevant_bom.product_uom_id._compute_quantity(qty_delivered, order_line.product_uom)
```

Comment:
Just asking, why does this need to be incremented?

Reply:
If we just assign a new value instead of incrementing, we override the value previously calculated in this `super(SaleOrderLine, self)._compute_qty_delivered()`